packages/api/src/qm/question.ts

```diff
@@ -97,7 +98,14 @@ export interface UserInputQuestion extends BaseQuestion {
   /**
    * question type
    */
-  type: "singleSelect" | "multiSelect" | "singleFile" | "multiFile" | "folder" | "text";
+  type:
+    | "singleSelect"
+    | "multiSelect"
+    | "singleFile"
+    | "multiFile"
+    | "folder"
+    | "text"
+    | "selectLocalFileOrInput";
```

Comment:
"singleSelectOrText" seems to be more consistant?

Reply:
guess you mean "singleFileOrText"?